<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>941</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>346</x>
     <y>270</y>
     <width>261</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #ffffff;
    background: transparent;
}</string>
   </property>
   <property name="text">
    <string>Forget Password?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>941</width>
     <height>751</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #2a1f4d,
        stop: 0.4 #3d2b6b,
        stop: 1 #1e1b2e
    );
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>256</x>
     <y>360</y>
     <width>136</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-size: 22px;
    font-weight: normal;
    color: #e0d4ff;
}</string>
   </property>
   <property name="text">
    <string>Email Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_edit">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>400</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>36</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: rgba(70, 60, 110, 0.8);
    color: #ffffff;
    border: 2px solid #6a5acd;
    border-radius: 22px;
    padding: 1px 150px 1px 50px;  /* جا برای آیکون */
    font-size: 18px;
}

QLineEdit:focus {
    border: 2px solid #9d81ff;
    background-color: rgba(90, 80, 140, 0.9);
}</string>
   </property>
   <property name="text">
    <string notr="true">Enter your Email</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="placeholderText">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="email_label">
   <property name="geometry">
    <rect>
     <x>645</x>
     <y>405</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../store-management-requirment/email_100dp_FFFFFF.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="send_email_Button">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>460</y>
     <width>450</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #6a5acd, 
        stop: 1 #9d81ff
    );
    color: white;
    border: none;
    border-radius: 22px;
    padding: 8px;
    font-size: 29px;
    font-weight: bold;
}

QPushButton:hover {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #9d81ff, 
        stop: 1 #b19bff
    );
    box-shadow: 0 0 20px rgba(157, 129, 255, 0.6);
}

QPushButton:pressed {
    background: qlineargradient(
        x1: 0, y1: 0, x2: 1, y2: 0,
        stop: 0 #5a4acd, 
        stop: 1 #8d71ef
    );
}</string>
   </property>
   <property name="text">
    <string>Send Email</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="back_login_Button">
   <property name="geometry">
    <rect>
     <x>396</x>
     <y>520</y>
     <width>134</width>
     <height>33</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton[flat=&quot;true&quot;] {
    background: transparent;
    color: #d8bfff;
    border: none;
    font-size: 22px;
}

QPushButton[flat=&quot;true&quot;]:hover {
    color: #ffffff;
    text-decoration: underline;
}</string>
   </property>
   <property name="text">
    <string>Back to Login</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="info_label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../store-management-requirment/info_outline_100dp_FFFFFF.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_3</zorder>
  <zorder>email_edit</zorder>
  <zorder>email_label</zorder>
  <zorder>send_email_Button</zorder>
  <zorder>back_login_Button</zorder>
  <zorder>info_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
